src/shared/sandboxCommandBase.ts

```diff
@@ -160,8 +160,8 @@ export abstract class SandboxCommandBase<T> extends SfCommand<T> {
       [
         messages.getMessage('sandboxSuccess'),
         messages.getMessages('sandboxSuccess.actions', [
```

Comment:
this msg is using a wrong flag:
https://github.com/salesforcecli/plugin-org/blob/main/messages/sandboxbase.md?plain=1#L8

`-e` doesn't exists in `org open`, that should be `--target-org`.

Reply:
@cristiand391 Good catch. Thank you